### PR TITLE
Improve returning member authentication UI

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Incremental improvements for returning member authentication
- Apply design to BankID login prompt
- Localize affected components

## Justify why they are needed

Make returning member flow look good. To be followed by other PRs with the same goal

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1525** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1525" target="_blank">Graphite</a> 
    * **PR #1529** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1529" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1529?utm_source=stack-comment).

File: apps/store/src/components/BankIdLogin.tsx [hunk 1]
```diff
@@ -1,7 +1,10 @@
 // TODO: Localize texts
 import { datadogLogs } from '@datadog/browser-logs'
+import styled from '@emotion/styled'
+import { useTranslation } from 'next-i18next'
 import { FormEventHandler, useState } from 'react'
-import { Button, Text } from 'ui'
+import { Button, Text, theme } from 'ui'
+import { BankIdIcon } from '@/components/CheckoutPage/BankIdIcon'
```
Comment (robinandeer) [inline apps/store/src/components/BankIdLogin.tsx:7]: nit: avoid importing scoped component helpers -- consider moving `BankIdIcon` to UI package or to a global component

File: apps/store/src/components/PriceCalculator/SsnSeSection.tsx [hunk 2]
```diff
@@ -105,18 +106,45 @@ const AuthenticationRequiredSsnSection = ({
...
+    </AuthPromptWrapper>
   )
 }
 
+const AuthPromptWrapper = styled.div({
+  position: 'absolute',
+  top: 0,
+  bottom: 0,
+  left: 0,
+  right: 0,
```
Comment (robinandeer) [inline apps/store/src/components/PriceCalculator/SsnSeSection.tsx:132]: nit: `inset: 0`

File: apps/store/src/components/PriceCalculator/SsnSeSection.tsx [hunk 3]
```diff
@@ -105,18 +106,45 @@ const AuthenticationRequiredSsnSection = ({
...
   return (
-    <div>
-      <Text>ssn: {ssn}</Text>
-      <Text>Looks like you are returning member. Login to pre-fill you information</Text>
-      <BankIdLogin shopSessionId={shopSessionId} ssn={ssn} onCompleted={onCompleted} />
-      <Button variant="ghost" onClick={onCompleted}>
-        Skip for now, login at checkout
-      </Button>
-    </div>
+    <AuthPromptWrapper>
```
Comment (robinandeer) [inline apps/store/src/components/PriceCalculator/SsnSeSection.tsx:112]: Hmmm why is this not a dialog? That means we need to manually worry about capturing focus, handle keyboard events, etc.

We can merge this now but I think we should add a comment that we need to address this in the future

**Review by robinandeer — APPROVED** [review]
Good stuff!

Left a comment about a11y support when not using radix dialog but think we can merge for now with a comment

File: apps/store/src/components/PriceCalculator/SsnSeSection.tsx (hunk 3 above)
Comment (alebedev) [reply apps/store/src/components/PriceCalculator/SsnSeSection.tsx:112]: Yes, this should be a dialog.  Will convert in separate PR

**Pull request merged**